Editor/EditorCore/UVEditor.cs

```diff
@@ -1535,6 +1535,10 @@ void RotateTool()
                         {
                             var uv = face.uv;
                             uv.rotation += uvRotation - t_uvRotation;
+                            if (uv.rotation > 360f)
+                                uv.rotation = uv.rotation % 360f;
+                            else if (uv.rotation < 0f)
+                                uv.rotation = 360f + (uv.rotation % 360f);
```

Comment:
Same note, is the intent to clamp between 0 and 360, or just prevent negative values?